examples/webgl_per_vertex_ao.html

```diff
@@ -0,0 +1,450 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - per vertex ao</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				font-family: Monospace;
+				color: #000;
+				margin: 0px;
+				overflow: hidden;
+			}
+			#info {
+				position: absolute;
+				top: 10px;
+				width: 100%;
+				text-align: center;
+				z-index: 100;
+				display:block;
+			}
+			#info a {
+				color: #046;
+				font-weight: bold;
+			}
+		</style>
+	</head>
+
+	<body>
+		<div id="info">
+			<a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> - per vertex ambient occlusion:
+			<a href="https://developer.nvidia.com/gpugems/GPUGems2/gpugems2_chapter14.html" target="_blank" rel="noopener">article</a><br>
+			<div id="performance"></div>
+		</div>
+
+		<script src="../build/three.js"></script>
+		<script src="js/libs/dat.gui.min.js"></script>
+
+		<script src="js/controls/OrbitControls.js"></script>
+
+		<script src="js/loaders/GLTFLoader.js"></script>
+
+		<script src="js/Detector.js"></script>
```

Comment:
`Detector` is deprecated. You definitely need to rebase your branch.

Reply:
Updated.